app/controllers/system/admin/lectures_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+class System::Admin::LecturesController < System::Admin::Controller
+  load_and_authorize_resource :lecture, class: System::Lecture.name
```

Comment:
So we also need a `System::Lecture` ?  system announcement is for announcing important stuff to entire coursemology.